legacy/firmware/fido2/resident_credential.c

```
@@ -0,0 +1,151 @@
+    cred_desc->cred_id_len = len - RP_ID_HASH_LENGTH;
+    memcpy(cred_desc->cred_id, cred_id_storage.credential_id,
+           cred_desc->cred_id_len);
+    if (ctap_authenticate_credential_data(cred_id_storage.rp_id_hash,
+                                          cred_desc) == 0) {
+      return 0;
+    }
+  }
+  return SE_FIDO2_SLOT_DATA_INVALID;
+}
+
+bool resident_credential_delete(uint8_t index) {
```

cppcheck (2.10-2)

[style] The function 'resident_credential_delete' is never used.

(unusedFunction)